docs/core/install/runtime.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Install .NET Core runtime on Windows, Linux, and macOS - .NET Core
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to run .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+ms.custom: "updateeachrelease"
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install the .NET Core Runtime
+
+In this article, you'll learn how to download and install the .NET Core runtime. The .NET Core runtime is used to run apps created with .NET Core.
+
+You can download and install .NET Core directly with one of the following links:
+
+- [.NET Core 3.1 Preview 3 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0)
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2)
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+
+## Install with an installer
+
+Both Windows and macOS have standalone installers that can be used to install the .NET Core 3.0 runtime.
+
+- Windows [x64 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-runtime-3.0.0-windows-x64-installer) | [x32 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-runtime-3.0.0-windows-x86-installer)
```

Comment:
> x32 CPUs

x86, 32-bit. The old website just said `x86`, if you want to clarify I'd go with something like `x86 CPUs (32-bit)`.

"x32 ABI" is a thing, but it isn't what that link is pointing to. 😄

Reply:
Thanks. That is just a mistake on my part.